src/main/features/response/views/confirmation/part-admission.njk

```diff
@@ -8,7 +8,7 @@
 
 {% if claim.response.paymentIntention %}
   {% if claim.response.paymentIntention.paymentDate %}
```

Comment:
@dharmendrak it seems pointless to have that 'if' statement when you do 'elseif claim.response.paymentIntention.paymentOption === domain.PaymentOption.BY_SPECIFIED_DATE' few lines down.